### PR TITLE
Copycode

### PR DESCRIPTION
**Pull request by dorianclay**
This feature isn't done yet, but I'm not sure how to properly subscribe to the `JoinGameResponse` to get what I'm looking for.

## Functionality should be as follows:
1. (User types into the `Join Game` input)
2. If game exists, route them to the `CreatePlayerComponent`
3. If no such game exists, stay on the create game page and notify user

### CODE REVIEW

**Comment by dorianclay** [comment]
Should be all working now, @ben-storyhealth, if you can take a look at this!

File: web/blackjack/src/app/game/create-game/create-game.component.ts [hunk 1]
```diff
@@ -1,27 +1,43 @@
...
-}
+  }
 
   onJoinGame(): void {
-    console.warn('Join game is not implemented yet.');
+    if (this.gameId == '') {
+      this.no_gameId = true;
+    } else {
+      this.blackjackService.joinGame(this.gameId).subscribe((response) => {
+        if (response.game_exists == false) {
```
Comment (ben105) [inline web/blackjack/src/app/game/create-game/create-game.component.ts]: For the comparison here, use three equal signs.

File: server/app.py [hunk 2]
```diff
@@ -33,6 +33,15 @@ def create_game():
     return {'game_id': game_id}
 
 
+@app.route('/blackjack/api/v1/games/<game_id>', methods=['POST'])
```
Comment (ben105) [inline server/app.py]: The POST verb is usually for sending data up to the server, with a post body. The HTTP method you probably want is GET.

File: web/blackjack/src/app/service/blackjack.service.ts [hunk 3]
```diff
@@ -2,11 +2,17 @@ import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs';
 import { environment } from 'src/environments/environment';
+// import { NetworkInterfaceBase } from 'node:os';
+// import * as internal from 'node:stream';
```
Comment (ben105) [inline web/blackjack/src/app/service/blackjack.service.ts]: I would remove commented code before submission.

**Review by ben105 — COMMENTED** [review]
Overall, looks good! Left a few comments, let me know what you think!

**Review by ben-storyhealth — APPROVED** [review]

**Pull request merged**